data/presets/emergency/fire_service_inlet.json

```diff
@@ -0,0 +1,22 @@
+{
+    "icon": "temaki-fire_hydrant",
+    "fields": [
+        "fire_mains"
```

Comment:
I think the `couplings` (https://wiki.openstreetmap.org/wiki/Key:couplings) field would be suitable for this preset as well.

Reply:
… and while we're at it maybe also `fire_sprinkler` as a checkbox field?

Reply:
> Maybe `ref` as well?

I don't know, if `ref` is used often? Maybe as optional tag.

Reply:
I was thinking about large buildings which might have more than one inlet, which might be numbered. Currently, 32 out of 1,300 features with `emergency=dry_riser_inlet` have a `ref`, so I guess an optional field would be more than sufficient.